composer.json

```diff
@@ -14,5 +14,10 @@
         "psr-4": {
             "App\\" : "src/"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "7.4"
```

Comment:
But here, we are telling composer to find packages that has the compatibility to the lower minimum PHP version so it can be compatible as long as possible.